Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/salvage.yml

```
@@ -274,12 +274,12 @@
       color: "#990000"
   - type: RandomSpawner
     rarePrototypes:
-    - MobRogueSiliconScrapThrustersRangedAsteroid
-    rareChance: 0.1
+    - MobSpawnScrapbotLaser #MobRogueSiliconScrapThrustersRangedAsteroid
```

```suggestion
    - MobSpawnScrapbotLaser
```
Are the comments really meaningful here?

---

Something-something, uhh